app/src/components/DataPanel.vue

```diff
@@ -54,6 +49,17 @@
               @update:comparelayertime="c => comparelayertime = c"
               @compareEnabled="compareEnabled = !compareEnabled"
             /> -->
+            <v-btn
```

Comment:
please remove all instances of commented out indicator-map (<!--<indicator-map>...)

Reply:
Fixed in [`3e0c603`](https://github.com/eurodatacube/eodash/pull/1516/commits/3e0c60337fc92fa77e8178edf286bdb6a904b55e)